packages/core/src/scripts/setupPrompts.ts

```diff
@@ -2,20 +2,30 @@ import { prisma } from "@oakai/db";
 
 import { PromptVariants } from "../models/promptVariants";
 import { lessonPlannerPrompts, quizGeneratorPrompts } from "../prompts";
+import { ailaGenerate } from "../prompts/lesson-assistant/variants";
 
 const main = async () => {
   try {
+    console.log("Setting up prompts");
+    console.log("Aila");
+    for (const variant of ailaGenerate.variants) {
+      console.log("variant", variant.slug);
+      const prompts = new PromptVariants(prisma, ailaGenerate, variant.slug);
+      await prompts.setCurrent(variant.slug, true);
+    }
+    console.log("Lesson Planner");
     for (const k of Object.keys(lessonPlannerPrompts)) {
       const prompt =
         lessonPlannerPrompts[k as keyof typeof lessonPlannerPrompts];
       const prompts = new PromptVariants(prisma, prompt, "main");
-      await prompts.setCurrent();
+      await prompts.setCurrent("main");
     }
+    console.log("Quiz Generator");
```

Comment:
can remove

Reply:
and other logs in this file?

Reply:
This gives you some feedback when you run pnpm prompts:dev, so you know what it's doing. I think that's helpful?